src/statsig/constants.ts

```diff
@@ -34,6 +34,7 @@ export const FeatureGates = {
   [StatsigFeatureGates.SHOW_NOTIFICATION_CENTER]: false,
   [StatsigFeatureGates.SHOW_CLOUD_ACCOUNT_BACKUP_SETUP]: false,
   [StatsigFeatureGates.SHOW_CLOUD_ACCOUNT_BACKUP_RESTORE]: false,
+  [StatsigFeatureGates.SHOW_MULTI_CHAIN_TRANSFERS]: false,
```

Comment:
it might be worth using a dynamic config so we can release multichain features one by one if we wanted, and have them all live in the same place.

as soon as one feature gate makes it into an app version we'll need to keep it, so I do think it makes sense to be forward-thinking on this.

Reply:
You mean [something like this](https://console.statsig.com/4plizaPmWwPL21ASV4QAO0/dynamic_configs/multi_chain_feature_gates)? A dynamic config that contains all the feature gates that we want?

Reply:
yes. you can probly get rid of the feature gate now

Reply:
ok now that I'm looking at this with "fresh eyes" (a weekend.. lol) I'm realizing that my suggestion is something we haven't done before, which is to use dynamic configs to essentially namespace our feature flags. if you're into this idea then go for it but I could imagine some arguments against doing it that way-- that it's an abuse of dynamic configs for instance, that feature gates are simpler, etc.

Reply:
the list of chains you set up is cool though

Reply:
Would having the dynamic config make rolling this out really complex? We'd likely roll out different features in different versions and the dynamic config would end up something like
```js
if (version === 'x') return { featureA: true, featureB: ['a'], ...}
if (version === 'y') return { featureA: false, featureB: ['a', 'b'], ... }
...
return { defaults }
```
Also, we won't be able to use metrics when rolling out features unless we use a feature gate itself as a condition in dynamic config, although not sure how important that is here.

But I do like that this allows us to use more than booleans.

Reply:
Yeah this is a good point -- I'll revert this back to using feature gates, since this seems like it adds more complexity than is worth it.